docs/en/ingest-management/release-notes/release-notes-8.4.asciidoc

```diff
@@ -22,11 +23,46 @@ Also see:
 * {kibana-ref}/release-notes.html[{kib} release notes]
 * {beats-ref}/release-notes.html[{beats} release notes]
 
+// begin 8.4.3 relnotes
+
+[[release-notes-8.4.3]]
+== {fleet} and {agent} 8.4.3
+
+Review important information about the {fleet} and {agent} 8.4.3 release.
+
+[discrete]
+[[bug-fixes-8.4.3]]
+=== Bug fixes
+
+{fleet}::
+* No bug fixes for this release
+
+{agent}::
+* Use at least warning level for all status logs {agent-pull}1218[#1218]
+* Fix unintended reset of source URI when downloading components {agent-pull}1252[#1252]
+* Create separate status reporter for local only events so that degraded
+{fleet} checkins no longer affect health on successful {fleet} checkins {agent-issue}1157[#1157] {agent-pull}1285[#1285]
+* Add success log message after previous checkin failures {agent-pull}1327[#1327]
```

Comment:
```suggestion
* Add success log message after previous checkin failures {agent-pull}1327[#1327]
* Fix Unix socket connection errors when using diagnostics command {agent-pull}2201[#2201]
```

There was no changelog entry for https://github.com/elastic/elastic-agent/pull/1034, so let's add this one.